src/main/java/com/jherrick/BasicBoardEvaluator.java

```diff
@@ -1,6 +1,30 @@
 package com.jherrick;
 
 public class BasicBoardEvaluator implements BoardEvaluator{
+    
+    private static int c0 = Constants.EVAL_CONSTANTS[0];
+	private static int c1 = Constants.EVAL_CONSTANTS[1];
+
+    public Board board;
+    public Side side;
+
+
+
+    public BasicBoardEvaluator(){
```

Comment:
I don't think these constructors are needed (or keeping Board and Side as instance variables). We should just pass the Board and the Side in when we want an evaluation.